docs/manual-EN/overview/data-model.md

```diff
@@ -0,0 +1,58 @@
+# Graph Data Modeling
+This guide is designed to walk you through the graph data modeling  of Nebula Graph. Basic concepts of designing a graph data model will be introduced.
+
+## Directed Property Graph
+
+The data model handled by the **Nebula Graph** is a **_directed property graph_**, whose edges are directional and there could be properties on both edges and vertices. It can be represented as:
+**G = < V, E, P<sub>V</sub>, P<sub>E</sub> >**
+
+Here **V** is a set of nodes aka vertices, **E** is a set of directional edges,
+**P<sub>V</sub>** represents properties on vertices, and **P<sub>E</sub>** is the properties on edges.
+
+## Tags
+In **Nebula Graph**, vertex properties are clustered by **tags**. A tag is like an entity type, which contains multiple properties. Each vertex can have multiple tags associated. There could be dependencies between tags, too.
+
+For instance, we could have a tag **Person**, which contains first_name, last_name, date_of_birth, etc. We could have another tag **Developer**, which has properties favorite_lang, github_account, etc. Tag **Developer** would depend on tag **Person**, which means if a vertex has tag **Developer** associated, it must have tag **Person** associated as well.
+
+## Vertices
+We will use the example graph below to introduce the basic concepts of the property graph:
+
+![image](https://user-images.githubusercontent.com/42762957/61120012-96b25a80-a4ce-11e9-8460-067cac52a1e0.png)
+
+Vertices are typically used to represent entities in the real world. In the preceding example, vertices are **player** and **team**. A vertex must be labeled with at least one tag. Each tag may have a set of properties called schema.
```

Comment:
IMO,  **player** and **team** are tags

Reply:
there are two kind of vertices labelled with tag **player** and **team**